dressForTheWeather/View Controllers/RecommendationViewController.swift

```diff
@@ -9,42 +9,54 @@
 import UIKit
 
 final class RecommendationViewController: UIViewController {
-    
+
     // MARK: - Properties
-    
+
     private var viewModel: RecommendationViewModel!
-    
+    private var backgroundGradientLayer: CAGradientLayer?
```

Comment:
I think the backgroundGradientLayer could be initialized like this:

```
private lazy var backgroundGradientLayer: CAGradientLayer = {
        let gradientLayer = CAGradientLayer()
        gradientLayer.frame = view.bounds
        gradientLayer.startPoint = CGPoint(x: 0, y: 0)
        gradientLayer.endPoint = CGPoint(x: 1, y: 1)
        return gradientLayer
    }()
```

Reply:
The compiler didn't let me use `lazy` for a computed property, but I was able to do this and not have it need to be optional anymore:

`private lazy var backgroundGradientLayer = CAGradientLayer()`

Change in a237f31cd0497c791d536454586543a504af8b2d